### ISSUE TITLE
`Search` border style

### ISSUE THREAD

**Issue by kaitoukid-1412** [EnixCoda/Gitako#282]
Hi

I think it looks better if the `left and right borders` `<Search/>` are removed

Before:

![image](https://user-images.githubusercontent.com/15698245/216650775-a62901b4-78ff-461c-8a20-9f050bd7718d.png)

After:

![image](https://user-images.githubusercontent.com/15698245/216650972-c3920df3-4ee9-4825-b241-9a5bee4cc935.png)

**Comment by EnixCoda** [comment]
Yes, indeed. Thanks for pointing it out, you have a good sense of design!

In fact, Gitako used to hide left & right borders exactly like the 2nd screenshot. But it was refactored to use primer components and then these styles are gone because I did not find a good way to style it like the old way.

I'll give another try if I got some free time.

---

Update: I gave a try on it and failed. The main blocker is handling the border style when search input was focused.

**Comment by kaitoukid-1412** [comment]
I really like your project

Have a nice day, @EnixCoda!
